docs/designers-developers/developers/tutorials/create-block/wp-plugin.md

```diff
@@ -78,13 +78,13 @@ To use the scripts package, the `scripts` property of package.json defines the p
   },
 ```
 
-These are run using: `npm run build` or `npm run start` which will call the appropriate binary and command we need.
+Sp these scripts are run by using: `npm run build` or `npm run start`
 
-Use `npm run build` for running once to create a production build.
+Use `npm run build` for running once to create a "production" build. This compresses the code down so it downloads faster, but makes it harder to read using browser tools—good for final deployment but not while developing.
 
-Use `npm run start` for creating a development build that also starts a watch process that waits and watches for changes to the file and will rebuild each time it is saved.
+Use `npm run start` for creating a "development" build, this does not compress the code so it is easier to read using browser tools. Additionally, development build will also start a watch process that waits and watches for changes to the file and will rebuild each time it is saved.
```

Comment:
`npm start` can be used but more importantly we can mention source maps that are used for debugging with some link for more details.

Reply:
Good suggestion, I added a link to MDN about using source maps.